ddtrace/appsec/iast/_util.py

```diff
@@ -1,18 +1,15 @@
 #!/usr/bin/env python3
-from functools import cache
```

Comment:
NB: I removed these because they log some deprecation warnings and that produces some test failures as the tests expect stdout and stderr to be empty.